src/integrationtest/java/com.microsoft.aad.msal4j/OAuthRequestValidationUnitT.java

```diff
@@ -7,8 +7,7 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
-import java.util.Collections;
-import java.util.Map;
+import java.util.*;
```

Comment:
I realize that intellij might do this by default, but most Java style guides that I've seen recommend against this. For example: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
there are different opinions about it, some do not like scrolling through import statements while browsing git hub, that is why it is default in Intellij, but if you like it better I will change settings

Reply:
I'd prefer not to include them. It's better to be explicit. 95% of the time I'm reading code in an IDE, not on GitHub.